### PR TITLE
WIX: allow user to customize install location

### PR DESCRIPTION
**Pull request by compnerd**
This change is [Reviewable](https://reviewable.io/reviews/compnerd/swift-build/309)

### CODE REVIEW

File: wix/windows-bundle.wxs [hunk 1]
```diff
@@ -17,22 +17,39 @@
...
         LogoFile="$(var.Logo)"
-        SuppressOptionsUI="yes"
+        SuppressOptionsUI="no"
         SuppressRepair="yes" />
     </BootstrapperApplicationRef>
 
+    <Variable
+      Name="INSTALLDIR"
+      Type="string"
+      Value="[WindowsVolume]"
```
Comment (wjk) [inline wix/windows-bundle.wxs:27]:
```suggestion
      Value="[WindowsVolume]\Library"
```

**Review by wjk — CHANGES_REQUESTED** [review]
I recommend that you change the default `INSTALLDIR` value from `[WindowsVolume]` to `[WindowsVolume]\Library`. If users choose to install the Swift runtime into a different folder, chances are it will be a newly created, otherwise-empty folder. This way, there won't be an otherwise-useless Library folder inside that folder.

Note that if you choose to do this, you will have to get rid of all the `<Directory Id="LIBRARY" Name="Library">` tags in the MSI projects to match the change in the bundle project. (Everything inside those tags can be kept as-is. Just delete the lines containing the opening and closing tags for the Library directory element, then auto-reindent.)

**Comment by compnerd** [comment]
Hmm, @wjk I think you have a good point.  However, I'm not convinced that should be part of this change.  Instead, we should make that change even if we don't actually manage to get this particular change merged because that actually improves the installer.

**Comment by wjk** [comment]
So you're saying that we should change the INSTALLDIR in a later PR? Your message is kinda confusing.

**Comment by compnerd** [comment]
I'm suggesting that we should do it in a different PR, but do that before this change.  I think that the change you are proposing will actually improve the installer for the extract only case and help with this PR.  Sorry about the message being confusing.

**Comment by compnerd** [comment]
I'm going to close this for now.  This is pretty out of date, and more importantly, the installer content has moved to https://github.com/apple/swift-installer-scripts.

**Pull request closed without merging**